packages/eslint-config-universe/README.md

```diff
@@ -1,5 +1,6 @@
 # eslint-config-universe
-Shared ESLint configs for Node, Web, and universal Expo projects.
+
+Shared ESLint configs for internal Expo projects.
```

Comment:
revert, I think this is used externally as well


```suggestion
Shared ESLint configs for Node, Web, and universal Expo projects.
```

Reply:
This change has already been accepted once by @ide, and was on the `main`.

AFAIK it's correct statement, and us not being prices in that matter lead to some issues/miscommunication in the past, and lead to creation of [`eslint-config-expo`](https://github.com/expo/expo/tree/main/packages/eslint-config-expo) which is tailored and recommended for external use with Expo.

Refs:
* https://exponent-internal.slack.com/archives/C1QP38NQ5/p1709744141238289?thread_ts=1709741770.192479&cid=C1QP38NQ5
* https://exponent-internal.slack.com/archives/C5ERY0TAR/p1637093462088600

Reply:
Ah, got it. Sounds good.